### ISSUE TITLE
Live counter to show characters remaining in Display Name and Bio fields on Edit Profile form

### ISSUE THREAD

**Issue by aral** [tootsuite/mastodon#1530]
<img width="417" alt="screen shot 2017-04-11 at 13 41 10" src="https://cloud.githubusercontent.com/assets/41798/24907793/e7e61232-1ebd-11e7-8ae0-42144f6de9a8.png">

The Edit Profile form doesn’t show a live counter on characters remaining for the display name and counter, thereby leaving it to the person to guess if they’re right and find out only after submitting.

To enforce the principle of “Prevent, Don’t Scold”, we should include a live character counter for both fields so you can see exactly how many characters you have remaining and also be prevented from submitting if you’re over the character limit.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

(I saw there was a similar issue but that its title and scope were then changed to increasing the character count instead, which is a related but separate issue.)

**Comment by rhaamo** [comment]
For me this is the same issue reported as #1447 even if it suggest a second thing (length extension).

**Comment by aral** [comment]
Hi @rhaamo, given that we can extend the length of the bio field to 500 characters and close #1447 without adding live counters to either the bio or the display name fields, I would respectfully stress that the two issues are not the same. If you’d like, I can add the live counter feature request to #1447 at which point the criterion for closing it would be the implementation of both the 500 character extension and the addition of live counters to the display name and bio fields but I feel it’s better if these were kept as two separate issues. Thoughts?

**Comment by wxcafe** [comment]
#1447 literally says <img height="20em" alt="screen shot 2017-04-11 at 18 00 08" src="https://cloud.githubusercontent.com/assets/3009327/24918448/bc88fba4-1ee0-11e7-92cc-3c4f9c523c5f.png"> in the title. I'm closing this as duplicate.
